setup.js

```diff
@@ -0,0 +1,22 @@
+// read env vars from local .env file
+const dotenv = require('dotenv');
+dotenv.config();
+
+// check for required env vars declared in app.json
+const fs = require('fs');
+const appJSON = fs.readFileSync('app.json', 'utf8');
+const appConf = JSON.parse(appJSON);
+
+let unset = false;
+Object.keys(appConf.env).forEach((key) => {
+  if (appConf.env[key].required && process.env[key] === undefined) {
```

Comment:
can we use default values here? I would rather not have to fill in all the vars in every environment, but still be able to control them